live_python_qa/utils.py

```diff
@@ -41,12 +41,12 @@ def estimateggdparam(vec):
     r_gam = (gamma(1.0/gam)*gamma(3.0/gam))/((gamma(2.0/gam))**2)
     # print(np.mean(vec))
     sigma_sq = np.mean(vec**2)  #-(np.mean(vec))**2
-    sigma = np.sqrt(sigma_sq)
+    #sigma = np.sqrt(sigma_sq)
```

Comment:
If sigma is not being used, let's remove these commented lines. Same on lines 42 and 43, and any other unnecessary comments.